sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -165,7 +168,8 @@ public PathogenTestForm addPathogenTestComponent(
 		CommitDiscardWrapperComponent<? extends AbstractSampleForm> sampleComponent,
 		PathogenTestDto pathogenTest,
 		int caseSampleCount,
-		Runnable callback) {
+		Runnable callback,
+		boolean isDeleteAllowed) {
```

Comment:
I find this parameter name a little misleading. If I set this to false, I would expect the pathogen test to not be deletable.
Yet for a user with the `UserRight.PATHOGEN_TEST_DELETE` right, it is. How about changing the name to isNew?

Also, I think it's a good idea to add some javadoc to this method.